libs/dh/metering-point/feature-search/src/lib/dh-metering-point.routes.ts

```diff
@@ -31,7 +31,7 @@ export const dhMeteringPointRoutes: Routes = [
   {
     path: '',
     canActivate: [
-      PermissionGuard(['fas']),
+      PermissionGuard(['metering-point:search']),
```

Comment:
Issue: Should it be both fas and metering-point:search that should have access?

Reply:
Good question. Let's discuss it tomorrow :)